src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/IssueCreateDialog.kt

```diff
@@ -49,13 +53,41 @@ internal class IssueCreateDialog(
         return this
     }
 
+    /**
+     * Click 'Configure Fields' to display 'All' to ensure
+     * all mandatory fields are displayed in creation dialog.
+     *
+     */
     fun fillRequiredFields(): IssueCreateDialog {
+        driver.wait(elementToBeClickable(configColumnField)).click()
+        val locator = By.xpath("//div[@id='inline-dialog-field_picker_popup']//a[text()='All']")
```

Comment:
We gotta let people `showAllFields()`, but I wouldn't force it within `fillRequiredFields`. Let's use it in HWR and when it's battle-tested enough, we can consider calling it here.

Reply:
Made the changes, but the travis CI failed, checking the CI locally